test/sqllogictest/transform/fold_vs_dataflow/4_text_folding.slt

```diff
@@ -25,18 +25,21 @@ SELECT * FROM t_data
 WHERE f1 ILIKE '%bCd%'
 ORDER BY f1;
 ----
+ABCDEF
 AbCdEf
+aBcDeF
+abcdef
 
 query T multiline
 EXPLAIN SELECT * FROM t_data
 WHERE f1 ILIKE '%bCd%';
 ----
 Explained Query:
-  Filter "%bCd%" ~~(#0)
-    Get materialize.public.t_data
+  Filter ilike["%bCd%"](#0)
```

Comment:
The `ilike` changes are fine: their EXPLAIN format was changed in https://github.com/MaterializeInc/materialize/pull/20602